### PR TITLE
Fixes for clang tidy warnings

### PR DESCRIPTION
**Pull request by Nagamani71**

### CODE REVIEW

File: ngraph_creator/operations/src/Strided_Slice.cpp [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        return false;
+    }
+
+    auto& strides_OperandIndex = sModelInfo->getOperationInput(mNnapiOperationIndex, 3);
+    if (!sModelInfo->isOperandLifeTimeConst(strides_OperandIndex)) {
+        ALOGE("%s Only Constant dimensions supported now", __func__);
+        return false;
+    }
+
+    auto shrink_axis_mask = sModelInfo->ParseOperationInput<int32_t>(mNnapiOperationIndex, 6);
```
Comment (anikulk) [inline ngraph_creator/operations/src/Strided_Slice.cpp:40]: Nit : Do you think when we have many indices in the operation we can avoid magic numbers by defining them?

File: ngraph_creator/operations/src/Strided_Slice.cpp [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    std::vector<int64_t> shrink_axis_mask_bits = getMaskBits(shrink_axis_mask, inputRank);
+
+    for (int i = 0; i < inputRank; i++) {
```
Comment (anikulk) [inline ngraph_creator/operations/src/Strided_Slice.cpp:43]: uint32_t

File: ngraph_creator/operations/src/Strided_Slice.cpp [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    std::shared_ptr<ngraph::Node> outputNode = std::make_shared<ngraph::opset3::StridedSlice>(
+        data, begin, end, strides, begin_mask_bits, end_mask_bits, new_axis_mask,
+        shrink_axis_mask_bits, ellipsis_mask);
+
+    return outputNode;
+}
+
+std::vector<int64_t> Strided_Slice::getMaskBits(int32_t maskValue, size_t vec_size) {
+    std::vector<int64_t> mask_bits(vec_size);
+    int i = 0;
```
Comment (anikulk) [inline ngraph_creator/operations/src/Strided_Slice.cpp:96]: uint32_t

File: ngraph_creator/operations/src/Strided_Slice.cpp [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+    std::shared_ptr<ngraph::Node> outputNode = std::make_shared<ngraph::opset3::StridedSlice>(
+        data, begin, end, strides, begin_mask_bits, end_mask_bits, new_axis_mask,
+        shrink_axis_mask_bits, ellipsis_mask);
+
+    return outputNode;
+}
+
+std::vector<int64_t> Strided_Slice::getMaskBits(int32_t maskValue, size_t vec_size) {
+    std::vector<int64_t> mask_bits(vec_size);
```
Comment (anikulk) [inline ngraph_creator/operations/src/Strided_Slice.cpp:95]: Can mask be a negative value?

**Comment by Nagamani71** [comment]
@anikulk this PR includes on "Fix for clang tidy warnings " changes. The other one came as part of merge conflict which is developed by Lakshmi and also merged to branch

**Comment by anikulk** [comment]
@Nagamani71 But it is part of this PR you are committing correct? @lakshmishreec Please address the comments.

**Comment by Nagamani71** [comment]
@anikulk no, that code is already merged. My changes were made before that commit, while merging my changes this "Strided_slice" commit got added after resolving merge conflicts. Only "Fix for clang tidy warnings " are the new changes.

**Comment by anikulk** [comment]
@Nagamani71 can you rebase your change. thx.

**Comment by Nagamani71** [comment]
Raised for 1.3 branch

**Comment by anikulk** [comment]
@RavirajSitaram Can we delete the 1.2 branch?

**Pull request closed without merging**